wp-content/plugins/goonj-blocks/src/render.php

```diff
@@ -24,6 +24,15 @@
 	$action_target['id'],
 );
 
+$pu_visit_check_link = sprintf(
+	'/office-visit/?target_id=%s',
```

Comment:
I don't undertand this link structure. All these actions are happening inside processing center so why are we not doing:
```suggestion
	'/processing-center/office-visit/?target_id=%s',
```

Reply:
I previously attempted to create the URL `processing-center/office-visit`, but when updating the URL in the pages, it automatically changes to `processing-center-office-visit`, preventing the desired format.